eservices-monitoring-exporter/src/main/scala/it/pagopa/interop/eservicesmonitoringexporter/Main.scala

```diff
@@ -0,0 +1,77 @@
+package it.pagopa.interop.eservicesmonitoringexporter
+
+import com.typesafe.scalalogging.Logger
+import it.pagopa.interop.commons.logging._
+import com.typesafe.scalalogging.LoggerTakingImplicit
+import it.pagopa.interop.commons.files.service.FileManager
+import it.pagopa.interop.commons.cqrs.model.ReadModelConfig
+import it.pagopa.interop.commons.cqrs.service.{MongoDbReadModelService, ReadModelService}
+import it.pagopa.interop.commons.utils.CORRELATION_ID_HEADER
+import it.pagopa.interop.eservicesmonitoringexporter.util.Jobs
+import it.pagopa.interop.eservicesmonitoringexporter.util.Utils._
+import it.pagopa.interop.eservicesmonitoringexporter.util.Configuration
+
+import java.util.concurrent.{ExecutorService, Executors}
+import scala.concurrent.duration.Duration
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Await, ExecutionContext, Future}
+
+import java.util.UUID
+import scala.util.Failure
+
+object Main extends App {
+
+  val logger: LoggerTakingImplicit[ContextFieldsToLog] =
+    Logger.takingImplicit[ContextFieldsToLog](this.getClass.getCanonicalName())
+
+  implicit val context: List[(String, String)] = (CORRELATION_ID_HEADER -> UUID.randomUUID().toString()) :: Nil
+
+  logger.info("Starting eservices monitoring exporter job")
+
+  def getFileManager(configuration: Configuration): Future[(FileManager, ExecutorService)] = Future {
+    val blockingThreadPool: ExecutorService = Executors.newFixedThreadPool(Runtime.getRuntime.availableProcessors())
+    (
+      FileManager.get(configuration.storage.kind match {
+        case "S3"   => FileManager.S3
+        case "file" => FileManager.File
+        case _      => throw new Exception("Incorrect File Manager")
+      })(ExecutionContext.fromExecutor(blockingThreadPool)),
+      blockingThreadPool
+    )
+  }
```

Comment:
maybe in this way is a bit clearer?
```suggestion
  def getFileManager(configuration: Configuration): Future[(FileManager, ExecutorService)] = Future {
    implicit val blockingThreadPool: ExecutorService = Executors.newFixedThreadPool(Runtime.getRuntime.availableProcessors())
    val fileManager =  FileManager.get(configuration.storage.kind match {
        case "S3"   => FileManager.S3
        case "file" => FileManager.File
        case _      => throw new Exception("Incorrect File Manager")
      }   
    (fileManager,blockingThreadPool)
  }
```

Reply:
It is right this?
def getFileManager(configuration: Configuration): Future[(FileManager, ExecutorService)] = Future {
    implicit val blockingThreadPool: ExecutorService =
      Executors.newFixedThreadPool(Runtime.getRuntime.availableProcessors())
    implicit val ec = ExecutionContext.fromExecutor(blockingThreadPool)  
    val fileManager                                  = FileManager.get(configuration.storage.kind match {
      case "S3"   => FileManager.S3
      case "file" => FileManager.File
      case _      => throw new Exception("Incorrect File Manager")
    })(ec)
    (fileManager, blockingThreadPool)
  }